### PR TITLE
Address CVEs

### LINKED ISSUE #87: Address CVE-2022-23648
The following CVE has been reported against the repo.

CVE-2022-23648

### PR DESCRIPTION
**Pull request by joe-atzinger**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:
Addresses CVEs.

**Which issue(s) this PR fixes**:
Fixes #87
Fixes #88

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,23 +6,18 @@ require (
...
 	github.com/pelletier/go-toml v1.9.4
 	github.com/stretchr/testify v1.8.0
 	golang.org/x/net v0.4.0
 	google.golang.org/grpc v1.42.0
-	k8s.io/api v0.24.3
 	k8s.io/apimachinery v0.24.3
-	k8s.io/client-go v0.24.3
 	k8s.io/klog/v2 v2.70.1
-	k8s.io/kubernetes v1.23.3
+	k8s.io/kubernetes v1.20.6
```
Comment (vinli-cn) [inline go.mod:20]: We need to upgrade k8s to 1.26.0

**Review by vinli-cn — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (joe-atzinger) [reply go.mod]: Good catch.  Updates.

File: go.mod [hunk 2]
```diff
@@ -17,7 +17,7 @@ require (
 	google.golang.org/grpc v1.42.0
 	k8s.io/apimachinery v0.24.3
 	k8s.io/klog/v2 v2.70.1
-	k8s.io/kubernetes v1.20.6
+	k8s.io/kubernetes v1.26.0
```
Comment (vinli-cn) [inline go.mod:20]: you need to rerun "go mod tidy" & "go mod vendor" again after you modify the go.mod

I'd recommend you revert all your commits in this branch and do this again.

Comment (joe-atzinger) [reply go.mod:20]: Thanks for catching it.  Reverted and (hopefully) got it right this time.

**Comment by fuxiuyin** [comment]
I have verified that the new code can be built in our pipeline.

/lgtm

**Review by fuxiuyin — APPROVED** [review]

**Pull request merged**